sdk/storage/azure-storage-common/inc/azure/storage/common/internal/xml_wrapper.hpp

```diff
@@ -37,44 +37,39 @@ namespace Azure { namespace Storage { namespace _internal {
 
   class XmlReader final {
```

Comment:
Based on the CI errors, this file needs memory to declare std::unique_ptr.

Reply:
Strange, it compiles for me locally, but my setup is also a bit weird (I compile the Azure submodule in ClickHouse).

Added `<memory>`.